### PR TITLE
Updated Vendor from type:string to string[]

### PR DESCRIPTION
**Pull request by kevincasey1222**
Hello @aiwilliams ! Per our discussion on Slack, this is a change to update the Vendor schema from a string type to an array of strings. Thanks!

### CODE REVIEW

File: src/schemas/Vendor.json [hunk 1]
```diff
@@ -11,31 +11,34 @@
...
-            "generic-service-provider",
-            "generic-subscription",
-            "CSP",
-            "ISP",
-            "MSP",
-            "MSSP",
-            "IdP",
-            "other"
-          ]
+          "type": "array",
```
Comment (austinkelleher) [inline src/schemas/Vendor.json:14]: Changing this to an array is a breaking change. Should we consider making this support both `string` and `array`?

**Review by austinkelleher — COMMENTED** [review]
I like the idea of supporting an array `category`, but I think we should consider making this `string | string[]`. Thoughts?

File: src/schemas/Vendor.json (hunk 1 above)
Comment (kevincasey1222) [reply src/schemas/Vendor.json:14]: @austinkelleher , totally agree. Do you know an existing object in the schema that has a property that can be both, so I can make sure my syntax is right?

Comment (kevincasey1222) [reply src/schemas/Vendor.json]: Based on Host.publicIpAddress, I tried this out.... does that look right?

Comment (aiwilliams) [reply src/schemas/Vendor.json]: I recommend adding a test! https://github.com/JupiterOne/data-model/blob/master/src/validateEntityWithSchema.test.ts

Comment (kevincasey1222) [reply src/schemas/Vendor.json]: Done!

**Pull request merged**